sdk/provisioning/Azure.Provisioning.Storage/tests/StorageTests.cs

```diff
@@ -39,6 +39,26 @@ public async Task StorageBlobDefaults()
             await ValidateBicepAsync();
         }
 
+        [RecordedTest]
+        public async Task StorageFileDefaults()
```

Comment:
Can you also update the ExistingResources test to cover the new resource?

Reply:
You will need to push the new test recordings - https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/core/Azure.Core.TestFramework#pushing-session-records-and-updating-assetsjson

Reply:
done